src/main/features/response/views/eligibility/how-much-paid-claimant.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t('Money claim') %}
+
+{% set heading = t('How much have you paid the claimant?') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="Response: How much have you paid the claimant option" data-event-label-from="option" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        <p>{{ t('The amount claimed is {{ totalAmountTillToday }}', { totalAmountTillToday: claim.totalAmountTillToday | numeral }) }}</p>
+
+
+        <legend class="visually-hidden">{{ t('How much have you paid the claimant?') }}</legend>
```

Comment:
doesn't radioGroup add its own legend? so wouldn't this end up adding two legends with the exact same text?
REQ

Reply:
Very good spot.